sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckstyleRegressionTest.java

```diff
@@ -36,7 +36,10 @@
 
     /** List of checks to suppress if we dynamically add it to the configuration. */
     private static final List<String> ADD_CHECK_SUPPRESSIONS = Arrays
-            .asList("ReturnCountExtendedCheck");
+            .asList(
+                    "EnumTrailingCommaCheck",
+                    "ReturnCountExtendedCheck"
```

Comment:
please add comment above "// initial ",
we will update it ones your PR is merged.
@rnveach , do you remember a reason ?

Reply:
@romani The reason was probably because it wasn't enabled in main repo as this is regression script.
See https://github.com/checkstyle/checkstyle/issues/3496 where it was enforced.
This line can probably be removed.

Reply:
See [my comment on the above review note](https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/709#discussion_r225831838): I have only added the comment to the line for `EnumTrailingCommaCheck`. I don't think touching the code regarding `ReturnCountExtendedCheck` is in the scope of this PR.

Reply:
ok, look ok now , I will update reason for second issue , ones we merge you.